### ISSUE TITLE
Http Trigger timeout after 230 seconds

### ISSUE THREAD

**Issue by dtfrt** [Azure/azure-functions-host#3086]
Hi everyone,

I am using Function App version 2.0-beta on consumption plan. The function is developed on C# VS2017, Microsoft.NET.Sdk.Functions - v1.0.13.

When I deploy the function to Azure Portal and create a request to it, the function timeouts after 230 seconds.

The host.json file contains:
{
  "functionTimeout": "00:10:00"
}

When I run the function locally, it passes the interval of 230 seconds and returns the response after 5 min.

Does anyone know how to solve this issue?

Thanks a lot!

**Comment by JohnReeson** [comment]
Azure function is one kind of Azure Web App, see this [SO thread](https://stackoverflow.com/questions/38673318/azure-asp-net-webapp-the-request-timed-out).

**Comment by dtfrt** [comment]
Depending from this reference: [functions-host-json](https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json)

We can modify the timeout for functions from 1 sec to 10 min (for Consumption Plan).
Is this still valid?

**Comment by JohnReeson** [comment]
@dtfrt Yes, but as for http trigger, it should return response in 230s, it's a request time out setting for Azure web app. Have a look at [durable function](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-overview) if you need to overcome this by design setting.
